package.json

```diff
@@ -33,6 +33,7 @@
     "@pika/pack": "^0.5.0",
     "@pika/plugin-build-node": "^0.9.2",
     "@pika/plugin-ts-standard-pkg": "^0.9.2",
+    "@types/aws-lambda": "^8.10.83",
```

Comment:
I think the types need to be added to `dependencies` as `aws-lambda` is a production dependency. Otherwise builds of TS projects that depend on it will start failing.

Reply:
Done. Thanks!